### PR TITLE
Fix paste in post title in GB-mobile.

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

Updates the title component to support copy/paste using the new RichText component.

## How has this been tested?

This can be test in GB-mobile using this PR:https://github.com/wordpress-mobile/gutenberg-mobile/pull/1575

## Types of changes

Bug fix
## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: packages/editor/src/components/post-title/index.native.js [hunk 1]
```diff
@@ -43,6 +43,20 @@ class PostTitle extends Component {
 		this.props.onSelect();
 	}
 
+	onPaste( { value, onChange, html, plainText } ) {
+		const content = pasteHandler( {
+			HTML: html,
```
Comment (mkevins) [inline packages/editor/src/components/post-title/index.native.js]: Should we remove this line, or set html to undefined or null? When I tried pasting text with links, the links were showing up as blue in the title :thinking: WDYT?

**Review by mkevins — COMMENTED** [review]
Hi @SergioEstevao :wave: 

Tested this on Android emulator (Pixel 3 XL, Android 9).

Pasted links are showing up as blue in the title.. other than that, this looks good!

File: packages/editor/src/components/post-title/index.native.js (hunk 1 above)
Comment (SergioEstevao) [reply packages/editor/src/components/post-title/index.native.js]: Good catch, going to remove it!

**Review by mkevins — APPROVED** [review]
Nice work @SergioEstevao . LGTM! :shipit:

**Pull request merged**